CODEOWNERS

```diff
@@ -0,0 +1,11 @@
+# Being an admin also gives you global code ownership.
+*                   @google/go-tpm-admin
+
+# Don't add all the admins by default for most things.
+# Note that any admins can still approve any PR.
+*                   @josephlr @awly @twitchy-jsonp
```

Comment:
This is a subset of go-tpm-admin.
Can we just remove this line?

Reply:
That line doesn't change ownership, but it does change the default attention set on a PR. This prevents every admin from getting automatically added to every PR.

See: https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax